src/test/java/io/vertx/ext/eventbus/bridge/tcp/TcpEventBusBridgeTest.java

```diff
@@ -210,6 +210,39 @@ public void testSendMessageWithReplyBacktrack(TestContext context) {
     });
   }
 
+  @Test
+  public void testSendMessageWithReplyBacktrackTimeout(TestContext context) {
+    // Send a request and get a response
+    NetClient client = vertx.createNetClient();
+    final Async async = context.async();
+
+    // This does not reply and will provoke a timeout
+    vertx.eventBus().consumer("test", (Message<JsonObject> msg) -> { /* Nothing! */ } );
+
+    client.connect(7000, "localhost", conn -> {
```

Comment:
use ctx.asyncAssertSuccess instead that avoids the `context.assertFalse(conn.failed())`

Reply:
Done, and everywhere else in this file, I did the modification.